src/sdk.js

```diff
@@ -0,0 +1,72 @@
+import Web3 from 'web3';
+import IPFSClient from 'ipfs-http-client';
+import url from 'url';
+import RegistryNetworks from 'singularitynet-platform-contracts/networks/Registry.json';
+import RegistryAbi from 'singularitynet-platform-contracts/abi/Registry.json';
+
+import Account from './Account';
+import MPEContract from './MPEContract';
+import ServiceClient from './ServiceClient';
+import { find } from 'lodash';
+
+const DEFAULT_GAS_LIMIT = 210000;
+const DEFAULT_GAS_PRICE = 4700000;
+
+export default class SnetSDK {
+  constructor(config) {
+    this._config = config;
+    const options = {
+      defaultGas: this._config.defaultGasLimit || DEFAULT_GAS_LIMIT,
+      defaultGasPrice: this._config.defaultGasPrice || DEFAULT_GAS_PRICE,
+    };
+    this._networkId = config.networkId;
+    this._web3 = new Web3(config.web3Provider, null, options);
+    this._mpeContract = new MPEContract(this._web3, this._networkId);
+    this._account = new Account(this._web3, this._networkId, this._config, this._mpeContract);
+    this._web3.eth.defaultAccount = this._account.address;
+    const registryAddress = RegistryNetworks[this._networkId].address;
+    this._registryContract = new this._web3.eth.Contract(RegistryAbi, registryAddress, { from: this._account.address });
+  }
+
+  get account() {
+    return this._account;
+  }
+
+  get mpeContract() {
+    return this._mpeContract;
+  }
+
+  get web3() {
```

Comment:
Should it be part of the `Identity` instead?

Reply:
Actually I cannot find if this method is really used anywhere

Reply:
The reason why we are exposing this is we the developer would need access to the `web3` instance so that she could operations like `getting` the current block number in the `PaymentChannelManagmentStrategy`. At the moment the `ServiceClient` is using it for a couple of things like [this](https://github.com/singnet/snet-sdk-js/pull/1/files?file-filters%5B%5D=.js#diff-622caf8e642effe65230813d19935735R133) and [this](https://github.com/singnet/snet-sdk-js/pull/1/files?file-filters%5B%5D=.js#diff-622caf8e642effe65230813d19935735R48).

Reply:
Added answer [here](https://github.com/singnet/snet-sdk-js/pull/1#discussion_r284163427)